src/app/core/entity/model/entity.ts

```diff
@@ -183,8 +183,9 @@ export class Entity {
    *
    * @returns {string} the unique id of this entity
    */
-  public getId(): string {
-    return this.entityId;
+  public getId(withPrefix: boolean = false): string {
```

Comment:
General question that I had during reading this PR. Is there any scenario in which we do **not** want to have the prefix stored? I cannot think of any at the moment so we should probably remove that option.

However, as this might have some implications on existing data we can also move this to another issue.

Reply:
--> #1526